### PR TITLE
graph-cli  + graph-ts v 0.4.0

### PR DESCRIPTION
**Pull request by orenyodfat**
- use temporary patched zip version of graph-cli till it will be fixed : see https://github.com/graphprotocol/graph-cli/pull/165 

- update reputation code

- use node minimum version 10.13.0

### CODE REVIEW

File: test/integration/util.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/**
+ * @jest-environment node
```
Comment (tsuberim) [inline test/integration/util.ts]: What is this comment?

File: test/integration/util.ts [hunk 2]
```diff
@@ -8,19 +11,21 @@ process.env = {
 };
 
 import axios from "axios";
+import adapter from 'axios/lib/adapters/http';
```
Comment (tsuberim) [inline test/integration/util.ts]: why is this needed?

File: test/integration/util.ts [hunk 3]
```diff
@@ -8,19 +11,21 @@ process.env = {
...
 import * as Web3 from "web3";
 import * as HDWallet from "hdwallet-accounts";
 
+
+
 const { node_http, ethereum, test_mnemonic } = process.env;
 
-export async function query(q: string, maxDelay = 500) {
+export async function query(q: string, maxDelay = 3000) {
```
Comment (tsuberim) [inline test/integration/util.ts]: is this the minimum delay that works?

File: README.md [hunk 4]
```diff
@@ -8,6 +8,7 @@ DAOstack subgraph for [TheGraph](https://thegraph.com/) project.
 
 2. `npm install`
 3. `npm run configure:<development|mainnet>` - configure the project to use ganache or mainnet via infura.
+4. `docker-compose build` - to rebuild docker image
```
Comment (tsuberim) [inline README.md]: change: "to rebuild docker changes after adding/removing packages"

**Comment by tsuberim** [comment]
@orenyodfat Do you also want to rebuild the ganache image on travis every time?

**Comment by orenyodfat** [comment]
> @orenyodfat Do you also want to rebuild the ganache image on travis every time?

If we can avoid that (and keep the patch) so lets do it

File: README.md (hunk 4 above)
Comment (orenyodfat) [reply README.md]: fix at 
bb7b475

File: test/integration/util.ts (hunk 3 above)
Comment (orenyodfat) [reply test/integration/util.ts]: bb7b475

File: test/integration/util.ts (hunk 2 above)
Comment (orenyodfat) [reply test/integration/util.ts]: bb7b475

File: test/integration/util.ts (hunk 1 above)
Comment (orenyodfat) [reply test/integration/util.ts]: bb7b475
